lms/envs/common.py

```diff
@@ -434,6 +434,17 @@
     'ENABLE_FOOTER_MOBILE_APP_LINKS': False,
 
     # Let students save and manage their annotations
```

Comment:
You can delete this line.  It is no longer needed now that we have this annotated.

Reply:
@robrap Thank you for your comments!I'll be updating them and attending any other suggestion!